### ISSUE TITLE
Impact of IPR

### ISSUE THREAD

**Issue by codehag** [tc39/tg5#36]
This was brought up by Istvan, though I think he has something else in mind. It is not clear to me what the concrete study Istvan was proposing, he would need to clarify.

Basically, one area that would be interesting to review is how have recent IPR changes (royalty free policy, new copyright) impacted TC39. A cross-standards study of interest would be how does IPR facilitate or prevent certain behaviors, and in turn, certain decisions.

**Comment by ecmageneva** [comment]
True. I am viewing the IPR policies on a higher level (i.e. SDO) than TC39 (which is an SDO Technical Committee). The reason for this is simple. This is always for all SDOs a matter on the level of the SDO itself, in this case, the Ecma GA. 

Of course as such it has significant impact also on TC39. 

1) Patent policy:

Currently TC39 (as an entire TC) operates with the Ecma Royalty-Free Patent Policy Option. In order to do that, the decision of the Ecma GA was needed that decided that TC39 is a RF TC. 

Earlier on this was only granted for a TC39 TG level as RF TG, but on the request of TC39 this was extended to the entire TC39 (I think in 2018 - I have to check the exact date..). 

Even earlier - maybe 10 years ago, again I have to look up the exact date - on the decision of the Ecma GA we worked in TC39 with an "Experimental TC39 Royalty Free Patent Policy Option".  The reason for that was that we have worked out such an experimental policy with the Ecma IPR Group, saw that as all SDO Patent Policies it was not a 100% perfect solution (I will come to this later maybe in a different comment), nevertheless the question was that in spite of this with the concrete topic and practice of "ECMAScript Standardization" the policy would work or not. To make it short, a few years of practice with the experimental policy we had no "blocking problems", so after a few years we  said, "ok let us make this as a formal Ecma policy". However, we should not forget, that this situation may change in practice. This depends on the facts how Patent Offices will decide on possible future patent applications on JavaScript/ECMAScript and connected to that if Ecma TC39 members will always have consensus among them either not to try to patent anything on ECMAScript (or if so then and awarded only without asking for royalties). So far this worked well. 

a) So,the first study point could be to make an analysis of the RF patent policies among the SDO and relevant Fora and compare them. Fortunately, they are not many such policies. This is different for the RAND Patent policies, where practically all SDOs offer that policy actually defaults, in some cases as option (e.g. W3C).

b) After that comparison, one will find that the RF policies differ in details. The question will be "why" and next "would make it sense to try to harmonize them and if so, how"? This would be interesting because that would investigate how to use "Normative Referencing" among RF Patent Policy based SDOs in their standards. When would such "Normative Referencing" harmonize and when not can not? The same question is very important between "Normative Referencing" of standards between an SDO with RF patent Policy and another with RAND Patent policy? When would such "Normative Referencing" work and when not?

c) General recognition of "Normative References". So this is not only for situations under b) but also in general about "Normative References". When is a "Normative Referencing" possible and when not. I think here the WTO-like standardization criteria would come into play.

d) Could the Patent Policies (all types: RF and RAND) be improved to make them "more waterproofed" - in order to have less concrete problems with them? Difficult, for many-many years very little progress on this. Why, and how could be improved?    

Similar exporation needed for the following IPR policies, but I am stopping here for today....

2) Copyright Policy:
a) Text Copyright
b) Software copyright

3) Trademark Policy:

4) Trade Secrets:

**Comment by ecmageneva** [comment]
"a) So, the first study point could be to make an analysis of the RF patent policies among the SDO and relevant Fora and compare them. Fortunately, they are not many such policies. This is different for the RAND Patent policies, where practically all SDOs offer that policy actually defaults, in some cases as option (e.g. W3C)."

On this first point actually I tested out with a dialog with ChatGPT 4.0. This comparison should be something that can handled be well by AI. Knowing the subject reasonably well, I am confident to share the results with the group (see attached PDF file). I have selected the few organizations that looked relevant to me. As mentioned most SDO and Fora Patent policies follow the RAND/FRAND patent policy regime. This is not what seems to be the best for language standardization, because of Open Source aspects and also the fact that the subject space is not really a topic with full of patents. So in that case why not to target a RF patent policy?

What one can see that the patent policies of the reviewed organizations are not the same, but in their spirits they are very close to one other, and certainly one could say that there is a level of compatibility where the different RF patent policies can be regarded as compatible. So, the question is can we close part of topic b) on that? Sertainly not the one about "Normative Referencing", where especially the interrelation between standards made with RF and RAND/FRAND patent policies are interesting. But that in a different contribution.[Comparison of Royalty-Free Patent Policies of some SDOs.pdf](https://github.com/user-attachments/files/17572330/Comparison.of.Royalty-Free.Patent.Policies.of.some.SDOs.pdf)
